bin/test-launcher

```diff
@@ -75,31 +75,31 @@ def run_cmd(cmd, input_str=None, from_dir=None, verbose=None, dry_run=False,
 def parse_command_line(args, description):
 ###############################################################################
     parser = argparse.ArgumentParser(
-        usage="""\n{0} <executable>
+        usage="""\n{0} -- <executable> [<exe-args>]
 OR
 {0} --help
 
 \033[1mEXAMPLES:\033[0m
     \033[1;32m# Lightweight python wrapper for a test that handles thread binding.\033[0m
-    > {0} ./my_exec
+    > {0} -- ./my_exec
 
 """.format(pathlib.Path(args[0]).name),
         description=description,
         formatter_class=argparse.ArgumentDefaultsHelpFormatter
     )
 
-    parser.add_argument("-e","--executable", help="The name of the executable to run",required=True)
     parser.add_argument("-b","--buffer-output", action="store_true",
                         help="Buffer all out and print all at once. Useful for avoiding interleaving output when multiple tests are running concurrently")
     parser.add_argument("-p","--print-omp-affinity", help="Print threads affinity at runtime",action="store_true")
 
     return parser.parse_known_args(args[1:])
 
 ###############################################################################
-def run_test(executable,buffer_output,print_omp_affinity,exec_args):
+def run_test(buffer_output,print_omp_affinity,exec_args):
 ###############################################################################
 
-    print("execname: {}".format(executable))
+    expect(exec_args, "Nothing to run?")
+    print("execname: {}".format(exec_args[1]))
```

Comment:
I think I added this when I was debuggin. We can remove it, I think. `run_cmd` will already take care of printing what we're running.